cmd/eksctl-anywhere/cmd/downloadimages.go

```diff
@@ -99,3 +106,17 @@ func packagerForFile(file string) packager {
 		return tar.NewPackager()
 	}
 }
+
+func fetchReader(reader *manifests.Reader, includePackages bool) artifacts.Reader {
+	if includePackages {
+		return curatedpackages.NewPackageReader(reader)
+	}
+	return reader
+}
+
+func fetchManifestDownloader(downloadFolder string, includePackages bool) artifacts.ManifestDownloader {
+	if includePackages {
+		return oras.NewBundleDownloader(downloadFolder)
+	}
+	return &artifacts.Noop{}
```

Comment:
can this return nil instead of defining Noop struct and the [Noop SaveManifests](https://github.com/aws/eks-anywhere/pull/2021/files#diff-54b0899b811165e615e0f50eb79e464c7848b322bf4f7b55af21b2c1004498dfR49) on it?

Reply:
It can. Any specific reason?

Reply:
I personally prefer the noop vs nil
That way `artifacts.Download` doesn't need to worry about the manifest downloader being conditional

A better pattern could be to have a slice of "downloaders" and just include the ones that are necessary. But that requires a bigger refactor so I would vote to leave it for later.